src/domains/issue/repository/issue.repository.impl.ts

```diff
@@ -111,4 +111,42 @@ export class IssueRepositoryImpl implements IssueRepository {
       });
     });
   }
+
+  /**
+   * Updates the "isBlocked" status of an issue.
+   * @param input - An object containing the issue ID and the new "isBlocked" status.
+   * @returns A Promise that resolves to an IssueDetailsDTO object representing the updated issue details.
+   */
+  async updateIsBlocked(input: IssueAndIsBlocked): Promise<IssueDetailsDTO> {
```

Comment:
I don't think it's okay to create a repo method just to update isBlocked attr. Anyway, this will be changed when [TRI-51](https://linear.app/tricker-sirius/issue/TRI-51/implement-ticket-edition) can be done.

Reply:
Yes, I had my doubts when I created the function. Should I change it or will we wait to TRI-51?

Reply:
This method is okay for the scope of the ticket. TRI-51 is the one that is specific for updating a ticket. This method will be replaced then, no worries.